compiler/modules/types/list.bal

```diff
@@ -126,8 +160,61 @@ function listSubtypeIsEmpty(Context cx, SubtypeData t) returns boolean {
     return isEmpty;    
 }
 
+function memberAt(ListMemberType m, int i) returns SemType {
```

Comment:
The scope of this is the whole module: every module-level symbol in list.bal should start with `list`.